python/tests/test_account.py

```diff
@@ -1505,11 +1504,11 @@ def test_send_receive_locations(self, acfactory, lp):
         ac1._evtracker.get_matching("DC_EVENT_SMTP_MESSAGE_SENT")
 
         lp.sec("ac2: wait for incoming location message")
-        ac2._evtracker.get_matching("DC_EVENT_INCOMING_MSG")  # "enabled-location streaming"
+        # ac2._evtracker.get_matching("DC_EVENT_INCOMING_MSG")  # "enabled-location streaming"
 
         # currently core emits location changed before event_incoming message
         ac2._evtracker.get_matching("DC_EVENT_LOCATION_CHANGED")
-        ac2._evtracker.get_matching("DC_EVENT_INCOMING_MSG")  # text message with location
+        # ac2._evtracker.get_matching("DC_EVENT_INCOMING_MSG")  # text message with location
```

Comment:
I guess you know what you are doing here, again

Reply:
yes, it's enough to wait for location-changed event below